Source/control.cpp

```diff
@@ -2118,38 +2008,38 @@ void __cdecl DrawChr()
 		MY_PlrStringXY(258, 239, 301, a4, a5[0], 0);
 	v9 = plr[myplr]._pMagResist;
 	a5[0] = v9 != 0;
-	if ( v9 >= 75 )
+	if ( v9 < 75 )
 	{
-		a5[0] = 3;
-		sprintf(a4, "MAX");
+		sprintf(a4, "%i%%", v9);
 	}
 	else
 	{
-		sprintf(a4, "%i%%", v9);
+		a5[0] = 3;
+		sprintf(a4, "MAX");
 	}
 	ADD_PlrStringXY(257, 276, 300, a4, a5[0]);
 	v10 = plr[myplr]._pFireResist;
 	a5[0] = v10 != 0;
-	if ( v10 >= 75 )
+	if ( v10 < 75 )
 	{
-		a5[0] = 3;
-		sprintf(a4, "MAX");
+		sprintf(a4, "%i%%", v10);
 	}
 	else
 	{
-		sprintf(a4, "%i%%", v10);
+		a5[0] = 3;
+		sprintf(a4, "MAX");
 	}
 	ADD_PlrStringXY(257, 304, 300, a4, a5[0]);
 	v11 = plr[myplr]._pLghtResist;
 	a5[0] = v11 != 0;
 	if ( v11 >= 75 )
 	{
-		a5[0] = 3;
-		sprintf(a4, "MAX");
+		sprintf(a4, "%i%%", v11);
 	}
 	else
 	{
-		sprintf(a4, "%i%%", v11);
+		a5[0] = 3;
+		sprintf(a4, "MAX");

```

Comment:
This broke lightning resistance on the character screen - the comparison didn't get flipped like magic/fire but the strings did.

Reply:
Fix in latest nightly